include/oboe/AudioStream.h

```diff
@@ -579,6 +579,100 @@ class AudioStream : public AudioStreamBase {
         return oboe::Result::ErrorUnimplemented;
     }
 
+    /**
+    * Informs the framework of an upcoming increase in the workload of a graphics pipeline
```

Comment:
This call is not really specific to graphics. They were just focussed on graphics when they were writing the doc. Maybe say "workload of an audio callback".